### ISSUE TITLE
Broken code when an op_asgn node contains a string with interpolation and the string contains a linefeed.

### ISSUE THREAD

**Issue by vidarh** [mbj/unparser#350]
If you parse code that produces an op_asgn node, such as e.g. a statement containing "+=" or "&=" etc., and the right hand side of that contains a string, and the string contains interpolation *and* a linefeed, the output ends up replacing the right hand side with a "<<- HEREDOC" but never outputs the HEREDOC.

Simplest test case I've found:

```ruby
y += "#{42}\n"
```

Test program:

```ruby
require 'parser/current'
require 'unparser'

puts Unparser.unparse(Unparser.parse('y += "#{42}\n"'))
```

Output:

```ruby
y += <<- HEREDOC
```

**Comment by vidarh** [comment]
Found this "in the wild" when trying to parse toml-rb (lib/toml-rb/dumper.rb) which has a more complex use to test against.

**Comment by vidarh** [comment]
Looking at #249 it seems likely to be a similar or the same issue.

**Comment by vidarh** [comment]
Pull request: https://github.com/mbj/unparser/pull/351

**Comment by mbj** [comment]
@vidarh Fix released in v0.6.10.
